Create Azure Linux 3.0 custom Arm VM (#5)
odidev/arm-learning-paths: content/learning-paths/servers-and-cloud-computing/azure-vm/_index.md

---
title:  Transform AArch64 ISO to VHD and Launch an Azure Linux Arm64 VM

minutes_to_complete: 120  

who_is_this_for: This Learning Path helps developers and system administrators deploy custom or community AArch64 Linux distributions on Azure Arm64 VMs. It covers converting an iso to a vhd, uploading it to Azure, and launching a fully functional VM—ideal when official Arm images are unavailable or for testing custom OS builds.

learning_objectives:
    - Convert an AArch64 ISO image into a fixed-size VHD using QEMU tools.

>     - Upload the VHD to Azure and create a custom image using Azure Shared Image Gallery.

Author: ```suggestion
    - Upload the VHD to Azure and use the Azure Shared Image Gallery to create a custom image
```

>     - Deploy and access an Azure Linux Arm64 VM based on the custom image.

Author: ```suggestion
    - Use Azure CLI to create Azure Linux 3.0 VM for Arm, using the custom image from Azure SIG
```

prerequisites:
    - A [Microsoft Azure](https://azure.microsoft.com/) account with permission to create resources, including Cobalt 100 (Arm64) instances (Dpsv6).
    - A local Linux or macOS machine with [QEMU](https://www.qemu.org/download/) installed to emulate AArch64 and convert images.
    - An [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) installed and authenticated on your local machine.
    
author: Jason Andrews

### Tags
skilllevels: Advanced

> subjects: Linux, AArch64, Virtual Machines, QEMU, Azure CLI

Author: ```suggestion
subjects: Containers and Virtualization
```

cloud_service_providers: Azure

armips:
    - Neoverse N2

tools_software_languages:
  - QEMU

>   - Azure CLI

Author: ```suggestion
  - Azure
  - Azure CLI
```
